server/file_hooks.go

```diff
@@ -46,4 +46,5 @@ func (p *Plugin) FileWillBeUploaded(c *plugin.Context, fileInfo *model.FileInfo,
 			)
 		}
 	}
+	return nil, "" // <--- This is the new return line
```

Comment:
Can we remove the comment in this line? I wrote it in my example simply as a guide.

Reply:
Thanks for pointing that out :)